doc/release-notes.md

```diff
@@ -38,8 +38,82 @@ report issues about Windows XP to the issue tracker.
 Notable changes
 ===============
 
-Example item
---------------
+Segregated witness soft fork
+----------------------------
+
+Segregated witness (segwit) is a soft fork that, if activated, will
+allow transaction-producing software to separate (segregate) transaction
+signatures (witnesses) from the rest of the data in a transaction, and
+to allow miners to place those witnesses outside of the traditional
+block structure. This provides two immediate benefits:
+
+- **Elimination of malleability:** Segregating the witness allows both
+  existing software and upgraded software that receives transactions to
+  calculate the transaction identifier (txid) of segwit-using
+  transactions without referencing the witness. This solves all known
+  cases of unwanted third-party transaction malleability, which is a
+  problem that makes programming Bitcoin wallet software more difficult
+  and which seriously complicates the design of smart contracts for
+  Bitcoin.
+
+- **Capacity increase:** Moving witness data outside of the traditional
+  block structure (but still inside a new-style block structure) means
+  new-style blocks can hold more data than older-style blocks, allowing
+  a modest increase to the amount of transaction data that can fit in a
+  block.
+
+Segwit also simplifies the ability to add new features to Bitcoin and
+improves the efficiency of full nodes, which will help provide long-term
+benefits to Bitcoin users.
+
+Activation for the segwit soft fork is being managed using BIP9 versionbits.
+The earliest activation date is set to 15 November 2016 (UTC) and
```

Comment:
Nov 15th is the earliest date that signalling can start. Earliest activation is likely to be ~28 days/4032 blocks from the next retarget after 15th Nov (estimated to be 20th Nov). I would hazard to be so specific.